### DISCUSSION TITLE
Can we access $vuetify in Pinia?

### DISCUSSION THREAD

**Discussion by Sjoerd82** [vuejs/pinia#1055] [Help and Questions]
I use **`this.$vuetify.breakpoint.mobile`** on a lot of pages throughout my application, and would love to centralize it as a reusable computed variable. I recently have incorporated Pinia in my app. Would be great if this could be in a getter. Something like:

```
    getters: {
        isMobile: state => {
            return ????.$vuetify.breakpoint.mobile
        }
    },
```

However, I'm not sure if this.$vuetify is accessible here (so far was not able to get to it).
I'm on Vue2 (because of Vuetify not being ready for Vue3 yet) + the Composition API plugin for Vue2, in case that matters.

**Comment by cannap** [comment]
for vue2 you could create a global mixin and use a computed isMobile... return this.$vuetify...


mixins/isMobile.js

```js
export default {

computed: { 
isMobile() {
return this.$vuetify.breakpoint.mobile
} 
}
}

```
and in main.js or hoo ever its called
Register it after vuetify
App.mixin(isMobile)

**Reply by Sjoerd82** [reply]
Yeah, thanks for that. So I take it that Pinia has no access to the "$-strucutre" then?

Vue3 (to which a lot of us, including me will migrate to soon) does away with Mixins. And given that I now have a global store anyway on most routes, doing something like this in there sounded like a good idea. I guess we can always create a computed function in a composable, but it means another import (which is not a disaster per se...)
